spec/models/maths_and_physics/school_eligibility_spec.rb

```diff
@@ -64,6 +64,16 @@
         end
       end
 
+      context "and it is a alternative provision school" do
+        let(:school_attributes) { {school_type: :pupil_referral_unit, phase: :not_applicable} }
+        it { is_expected.to be false }
+
+        context "and it has a statutory high age of over 11" do
```

Comment:
How do you feel about these nested contexts in terms of how they document and communicate about the behaviour?

Reply:
When I looked at the school eligibility specs in the past, I wasn't able to get a sense of what all these different combinations of attributes meant, and whether they occur in real life. When I wrote further tests for school eligibility, I included an [example of a real-life school for each one](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/spec/models/student_loans/school_eligibility_spec.rb#L200). It might be worth doing here, too.